platform/admin/ui/Gruntfile.js

```diff
@@ -175,7 +180,7 @@ module.exports = function (grunt) {
     });
 
 
-    grunt.registerTask('test', ['port:allocator', 'express:test', 'mochaWebdriver:phantom']);
+    grunt.registerTask('test', [ 'simplemocha:test' /*, 'port:allocator', 'express:test', 'mochaWebdriver:phantom'*/ ]);
```

Comment:
I'm not sure whether we should delete them or comment them out.  I was under the impression someone would go back and fix the tests that have been broken, and then uncomment them.  But they could also just look at the file history and figure it out.  

Then again, we should consider moving to having mostly unit tests (like the one in this PR).  We could still have tests that involve the browser, but we shouldn't have too many since they're a lot more brittle and inconsistent (and slower).  If we go that route, we could scrap the old tests.